src/index.tsx

```diff
@@ -63,5 +66,7 @@ export {
   Palette,
   Typography,
   Elevation,
+  overrides,
```

Comment:
Do we need to expose this?

Reply:
in case we want to know overrides, i didnt had a case in mind but we expose all others for the same reasoning